Kernel/FileSystem/ProcFS.cpp

```diff
@@ -746,8 +746,8 @@ Optional<KBuffer> procfs$df(InodeIdentifier)
         fs_object.add("readonly", fs.is_readonly());
         fs_object.add("mount_flags", mount.flags());
 
-        if (fs.is_disk_backed())
-            fs_object.add("device", static_cast<const DiskBackedFS&>(fs).device().absolute_path());
+        if (fs.is_file_backed())
+            fs_object.add("file", static_cast<const FileBackedFS&>(fs).file_description().absolute_path());
```

Comment:
Either `device` or `file` too specific, so we might want to use a more general key name here.

Reply:
"File" sounds fine to me; but we might want to say "source" or "image".

Reply:
I thought about "source", since we provide a `source_fd` parameter to `mount()`.